scripts/package.lua

```diff
@@ -47,7 +47,7 @@
 		error("Unsupported host os", 0)
 	end
 
-	local usage = 'usage is: package <branch> <type> [<compiler>]\n' ..
+	local usage = 'usage is: package <branch> <type> [<compiler>] [force]\n' ..
 		'       <branch> is the name of the release branch to target\n' ..
 		'       <type> is one of "source" or "binary"\n' ..
 		'       <compiler> (default: ' .. allowedCompilers[1] .. ') is one of ' .. table.implode(allowedCompilers, "", "", " ")
```

Comment:
Missing doc/help for `force` flag.

Reply:
Good catch. I'll add that in my next commit. Still wrapping my head around how to get this thing working for PRs across repos.  It worked in my repo, but didn't work after I made the PR.

Reply:
Follow up. We have no documentation for this file at all. We can add it though if you think we need it.

Reply:
I meant `usage` string "explains" quickly `<branch>` `<type>` `<compiler>` but not `[force]` (as non interactive).
Not really important...

Reply:
Looks like the latest commit doesn't have this change @nickclark2016?